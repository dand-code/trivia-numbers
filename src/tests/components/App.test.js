import React from 'react';
import { create } from 'react-test-renderer';

function App(props) {
  return `<GlobalStyle />
  <MainWrapper>
  <Title>
    Trividado
  </Title>
  <Switch>
    <Route exact path="/" component={Welcome} />
    <Route path="/trivial" render={props => <Questions {...props} questions={questions} />} />
  </Switch>
</MainWrapper>`;
}

describe("APP component", () => {
  test("Matches the snapshot", () => {
    const app = create(<App />);
    expect(app.toJSON()).toMatchSnapshot();
  });
});