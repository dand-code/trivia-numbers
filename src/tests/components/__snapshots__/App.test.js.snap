// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`APP component Matches the snapshot 1`] = `
"<GlobalStyle />
  <MainWrapper>
  <Title>
    Trividado
  </Title>
  <Switch>
    <Route exact path=\\"/\\" component={Welcome} />
    <Route path=\\"/trivial\\" render={props => <Questions {...props} questions={questions} />} />
  </Switch>
</MainWrapper>"
`;
